src/models/Offer.js

```diff
@@ -45,7 +45,8 @@ const offerSchema = mongoose.Schema({
   }
 })
 
-offerSchema.methods.isExpired = function () {
+// Arrow functions don't work here for some reason
```

Comment:
They probably don't work, because otherwise `this` wouldn't refer to the Offer... Read more here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Description

Reply:
Yeah I think that might be the reason